packages/os/os.spec

```
@@ -423,6 +425,13 @@ Conflicts: %{_cross_os}bash
 %description -n %{_cross_os}brush
 %{summary}.
 
+%package -n %{_cross_os}whippet
+Summary: A simple dbus-broker launcher
```

nit: let's avoid "simple" as it's overly subjective
```suggestion
Summary: Custom launcher for the D-BUS message broker
```